events/events_handler.go

```diff
@@ -0,0 +1,309 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
nit: just handler.go/handler_test.go since we're already in the events directory